### PR TITLE
Provide a way to generate redirects for stale links

### PR DESCRIPTION
**Pull request by ethanbustad**
Let me know what you think!

### CODE REVIEW

File: site/docs/redirects.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+	"dxp/7.x/en/content-authoring-and-management/recycle-bin/user-guide/configuring-the-recycle-bin.html": "dxp/7.x/en/content-authoring-and-management/recycle-bin/configuring-the-recycle-bin.html"
```
Comment (rbohl) [inline site/docs/redirects.json:2]: :ship: 

If we can do this we should immediately add the link that's currently broken in portal:
```json
"dxp/7.x/en/content-authoring-and-management/tags_and_categories/user-guide/defining-categories-and-vocabularies-for-content.html": "dxp/7.x/en/content-authoring-and-management/tags-and-categories/defining-categories-and-vocabularies-for-content.html"
```

**Comment by jrhoun** [comment]
Forwarded here: https://github.com/brianchandotcom/liferay-learn/pull/127

File: site/docs/redirects.json (hunk 1 above)
Comment (jrhoun) [reply site/docs/redirects.json:2]: @rbohl - i'm going to commit this line today

Comment (rbohl) [reply site/docs/redirects.json:2]: Yes! When it goes live I'll test it out from a local bundle--look forward to being amazed

**Pull request closed without merging**